packages/google-tag-manager/index.js

```diff
@@ -10,6 +10,11 @@ module.exports = async function nuxtTagManager(_options) {
     query: {}
   })
 
+  // Don't include when run in dev mode
+  if (this.nuxt.options.dev && process.env.NODE_ENV !== 'production') {
```

Comment:
I would simplify this condition:

```js
if (this.options.dev) {
```

And the correct one is: `this.options` and not `this.nuxt.options`

Reply:
Nice catch ! I fixed it.